### PR TITLE
Add config option to manta modules

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/manta/somatic/main.nf
Comment (maxulysse) [inline modules/nf-core/manta/somatic/main.nf:1]: Can you reorganize the mv part of the script in a similar way?

**Comment by maxulysse** [comment]
LGTM, but no testing of this new feature?

**Comment by nvnieuwk** [comment]
> LGTM, but no testing of this new feature?

Oh right, I tested it locally but forgot I had to add some tests here too :grin:

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
what is the config you put in doing?

**Comment by nvnieuwk** [comment]
From the default config:
```
# Remote read retrieval is used to improve the assembly of putative insertions by retrieving any mate reads in remote
# locations with poor mapping quality, which pair to confidently mapping reads near the insertion locus. These reads
# can help to fully assemble longer insertions, under certain circumstances this feature can add a very large runtime
# burden. For instance, given the very high chimeric pair rates found in degraded FFPE samples, the runtime of the read
# retrieval process can be unpredictable. For this reason the feature is disabled by default for somatic variant calling.
# This feature can be enabled/disabled separately for germline and cancer calling below.
#
# Here "CancerCallingModes" includes tumor-normal subtraction and tumor-only calling. "GermlineCallingModes" includes
# all other calling modes.
enableRemoteReadRetrievalForInsertionsInGermlineCallingModes = 1
enableRemoteReadRetrievalForInsertionsInCancerCallingModes = 0
```

**Pull request merged**